src/components/CommentList/CommentList.jsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import Comment from '../Comment/Comment';
+
+const CommentList = ({ comments }) => (
+  <ul>
+    {comments.map(comment => <Comment {...comment} key={comment.id}/>)}
+  </ul>
+);
+
+export default CommentList;
```

Comment:
good job!)
add a new line after all exports)
and I think that its batter to export variables (CommentList, Post ...... ) not default;

Reply:
new line